tests/unit/s2n_record_size_test.c

```diff
@@ -251,6 +276,33 @@ int main(int argc, char **argv)
             EXPECT_LESS_THAN_EQUAL(bytes_written, RECORD_SIZE_LESS_OVERHEADS);
         }
 
+        /* TLS1.3 chacha20 */
+        if (s2n_chacha20_poly1305.is_available()) {
```

Comment:
This test seems like a duplicate of an existing test: 

 https://github.com/aws/s2n-tls/blob/395da6528e8758ca7e68f6647d8bafc93afb21d5/tests/unit/s2n_record_size_test.c#L230-L252
 
 Why does the diff show it as you are adding a new test here? I dont see the removal/modification to the older test.

Reply:
It's a new test. Like the comment says, it uses the TLS1.3 version of the chacha20 record alg. The existing test is for the TLS1.2 chacha20 record alg. Same for AEAD: TLS1.2 and TLS1.3 are different.

That's why this bug could exist. We were only testing the old record algorithms.